example/server.js

```diff
@@ -8,24 +8,18 @@ const webpackServeBundle = require('./webpackServeBundle');
 const app = express();
 const isDevelopment = process.env.NODE_ENV !== 'production';
 
+const middlewarePath = '/falcor-postman';
+const falcorModelPath = '/example.json';
+
+const options = { middlewarePath, falcorModelPath, app };
+
 if (isDevelopment) {
-  webpackServeBundle(app);
+  webpackServeBundle(options);
```

Comment:
hi @nickbalestra I am sorry but I really do not see the benefit of using this `webpackServeBundle` if is development; are you able to explain to me what it does?